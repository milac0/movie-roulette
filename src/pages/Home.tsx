import React from "react";
import { css } from "emotion";

interface Props {}

const Home: React.FC<Props> = () => {
  return <div>Home</div>;
};

export default Home;
