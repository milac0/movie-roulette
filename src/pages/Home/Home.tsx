import React, { Fragment } from "react";
import RouletteButton from "../../components/RouletteButton/RouletteButton";
import SearchBar from "../../components/SearchBar/SearchBar";
import FiltersBar from "../../components/FiltersBar/FiltersBar";
import MovieList from "../../components/MovieList/MovieList";
import LoadMoreButton from "../../components/LoadMoreButton/LoadMoreButton";

interface Props {}

const Home: React.FC<Props> = () => {
  return (
    <Fragment>
      <SearchBar />
      <FiltersBar />
      <MovieList />
      <LoadMoreButton />
      <RouletteButton />
    </Fragment>
  );
};

export default Home;
