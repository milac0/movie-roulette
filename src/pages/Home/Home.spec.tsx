import React from 'react';
import renderer from 'react-test-renderer';

import Home from './Home';

describe('Home renders correctly', () => {
    test('snapshot renders', () => {
        const tree = renderer.create(<Home />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});

