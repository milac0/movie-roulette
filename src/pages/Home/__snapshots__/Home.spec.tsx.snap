// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly snapshot renders 1`] = `
Array [
  <div
    className="searchbar"
  >
    <div
      className="container"
    >
      <form
        onSubmit={[Function]}
      >
        <input
          name="search"
          onChange={[Function]}
          placeholder="search for..."
          type="text"
          value=""
        />
        <button
          type="submit"
        >
          Search!
        </button>
      </form>
    </div>
  </div>,
  <div
    className="container"
  >
    <button
      className="activeFilter"
      onClick={[Function]}
      value="popular"
    >
      Popular
    </button>
    <button
      className={null}
      onClick={[Function]}
      value="now_playing"
    >
      Now Playing
    </button>
    <button
      className={null}
      onClick={[Function]}
      value="top_rated"
    >
      Top Rated
    </button>
    <button
      className={null}
      onClick={[Function]}
      value="upcoming"
    >
      Upcoming
    </button>
  </div>,
  <div
    className="list"
  />,
  <div
    className="loadMore"
  >
    <button
      onClick={[Function]}
    >
      load
    </button>
  </div>,
  <div>
    <img
      className="rouletteBtn"
      onClick={[Function]}
      src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNk+P+/HgAFhAJ/wlseKgAAAABJRU5ErkJggg=="
    />
  </div>,
]
`;
